src/core_modules/capture-core/components/ListView/index.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+export { default as EventList } from './ListWrapper/EventsListWrapper.container';
+export { actionTypes as paginationActionTypes } from './Pagination';
+export { actionTypes } from './listView.actions';
```

Comment:
What is the plan with the actions of this file? Are we gonna refactor in a way that this is being used in a future PR?

Reply:
Yes, this will be refactore and used in future PRs (I did update imports to use this export, but as stated in previous comment it doesn't fit together logically just yet).